index.bs

```diff
@@ -4437,6 +4439,252 @@ The [=remote end subscribe steps=], with [=subscribe priority=] 10, given
 
 </div>
 
+## Input ## {#module-input}
+
+The <dfn export for=modules>input</dfn> module contains functionality for
+simulated user input.
+
+### Definition ### {#module-input-definition}
+
+[=remote end definition=]
+
+<pre class="cddl remote-cddl">
+
+InputCommand = (
+  InputPerformActionsCommand //
+  InputReleaseActionsCommand
+)
+</pre>
+
+### Commands ### {#module-input-commands}
+
+#### The input.performActions Command ####  {#command-input-performActions}
+
+The <dfn export for=commands>input.performActions</dfn> command performs a
+specified sequence of user input actions.
+
+<dl>
+   <dt>Command Type</dt>
+   <dd>
+    <pre class="cddl remote-cddl">
+      InputPerformActionsCommand = {
+        method: "input.performActions",
+        params: InputPerformActionsParameters
+      };
+
+      InputPerformActionsParameters = {
+        context: BrowsingContext,
+        actions: [* InputSourceActions]
+      };
+
+      InputSourceActions = (
+        NoneSourceActions //
+        KeySourceActions //
+        PointerSourceActions //
+        WheelSourceActions
+      );
+
+      NoneSourceActions = {
+        type: "none",
```

Comment:
What are all the "none" bits used for? They only appear in the CDDL so far. Related to https://w3c.github.io/webdriver/#null-input-source?

Reply:
All this CDDL is just representing the data format that's already used in WebDriver where it's checked manually.